modules/matterhorn-scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1246,10 +1249,17 @@ public int compare(ARecord o1, ARecord o2) {
     for (final ARecord r : records) {
       final Date start = r.getProperties().apply(Properties.getDate(START_DATE_CONFIG));
       final Date end = r.getProperties().apply(Properties.getDate(END_DATE_CONFIG));
-      if (checkStart.equals(start) || checkEnd.equals(end) ||      //If the start or end times are identical (.after and .before are strict comparisons)
-          checkStart.after(start) && checkStart.before(end) ||     //If the potential event starts during event r
-          checkEnd.before(end) && checkEnd.after(start) ||         //If the potential event ends during event r
-          checkStart.before(start) && checkEnd.after(end)) {       //If the potential event begins before, and ends after event r (ie, containment)
+      /*
+      If the start or end times are identical (.after and .before are strict comparisons) OR
+      If the potential event starts during event r OR
+      If the potential event ends during event r OR
+      If the potential event begins before, and ends after event r (ie, containment)
+      */
+      if (checkStart.equals(start) || checkEnd.equals(end)
```

Comment:
This test now seems redundant: The eventWithinMinimumSeparation already does test this, doesn't it?